app/src/main/java/com/ustwo/boilerplate/base/BasePresenter.java

```diff
@@ -41,7 +40,6 @@ public void onViewWillShow(@NonNull final T view) {
      */
     @CallSuper
     public void onViewWillHide() {
```

Comment:
I think the callbacks are necessary since it's very common to perform business logic when the screen resumes (e.g. refresh data, check permissions).

So rather than remove altogether I'd be in favour of reinstating the disposables. @jarekankowski what do you think?